var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -238,3 +238,9 @@ def test(self):
 
     def install(self, spec, prefix):
         make('install')
+
+        if spec.satisfies('%fj'):
```

Comment:
@t-karatsu this looks like an excellent candidate for upstream inclusion in https://gitlab.kitware.com/cmake/cmake/
Do you want to submit a PR thereto, too? :)

Reply:
@ax3l Thanks for your comment !
I'd like this fix to be included in the cmake upstream as well, but we'll ask the Fujitsu compiler development responsibility team to submit it.
However, if that seems to take time, I'll request upstream to fix it.

Reply:
Thanks, @t-karatsu - sounds great!